Userland/Libraries/LibJS/Tests/builtins/Symbol/Symbol.js

```diff
@@ -17,3 +17,11 @@ test("constructing symbol from symbol is an error", () => {
         Symbol(Symbol("foo"));
     }).toThrowWithMessage(TypeError, "Cannot convert symbol to string");
 });
+
+test("setting new properties on a symbol is an error in strict mode", () => {
+    "use strict";
+    var symbol = Symbol("foo");
+    expect(() => {
+        symbol.bar = 42;
+    }).toThrow(TypeError, "Cannot set property 'bar' of Symbol(foo)");
```

Comment:
That's not the right one, you need to use `toThrowWithMessage`

Reply:
Ah thanks for the help, I'm totally a noob here 😅 

... doubly so, as this helped catch another latent bug: we were printing out the assigned value instead of the assignment target, i.e. `Cannot set property 'bar' of 42`